provisioning/datasources/aws-iot-twinmaker.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: 1
+
+deleteDatasources:
+  - name: AWS IoT TwinMaker
+    orgId: 1
+
+datasources:
+  - name: AWS IoT TwinMaker
+    type: grafana-iot-twinmaker-datasource
+    enabled: true
+    jsonData:
+      authType: keys
+      defaultRegion: us-east-1
+      assumeRoleArn: $ASSUME_ROLE_ARN
+      workspaceId: $WORKSPACE_ID
+    secureJsonData:
+      accessKey: $ACCESS_KEY
```

Comment:
Where do these values come from? are they just on the machine?

Reply:
We'll store them in vault and fetch them the workflow runs and them pass them as env vars